src/recipes/eas-update.ts

```diff
@@ -1,86 +1,59 @@
 import { REPOSITORY_SECRETS_HELP_URL } from '../constants'
-import { CycliRecipe, CycliToolbox, ProjectContext, RunResult } from '../types'
+import { CycliRecipe, CycliToolbox, ProjectContext } from '../types'
 import { join } from 'path'
 
 const FLAG = 'eas-update'
 
-const execute =
-  () => async (toolbox: CycliToolbox, context: ProjectContext) => {
-    toolbox.interactive.vspace()
-    toolbox.interactive.sectionHeader(
-      'Genereating EAS Update and Preview workflow'
-    )
-
-    toolbox.dependencies.add('expo', context)
-
-    if (toolbox.filesystem.exists('eas.json')) {
-      toolbox.interactive.step(
-        'Detected eas.json, skipping EAS Build configuration.'
-      )
-    } else {
-      await toolbox.interactive.spawnSubprocess(
-        'EAS Build configuration',
-        'eas build:configure -p all'
-      )
-
-      toolbox.interactive.step('Created default EAS Build configuration.')
-    }
-
-    await toolbox.interactive.spawnSubprocess(
-      'EAS Update configuration',
-      'eas update:configure'
-    )
-
-    await toolbox.workflows.generate(join('eas', 'eas-update.ejf'), context)
+const execute = async (
+  toolbox: CycliToolbox,
+  context: ProjectContext
+): Promise<void> => {
+  toolbox.interactive.vspace()
+  toolbox.interactive.sectionHeader(
+    'Genereating EAS Update and Preview workflow'
```

Comment:
In general, I would call this step Preview with/using EAS Update, I have a feeling that when people don't know EAS Update may skip it after seeing the name, and Preview may catch their attention, wdyt?

Reply:
I agree, although I think that we also should add link to EAS Update docs and maybe some kind of demo how the preview looks like (the comment with QR code)?

Reply:
changed to "Preview with EAS Update" in 9c1def7afe372e509ae91a2ab9f59e75535a1bd8